src/lib/invite.js

```diff
@@ -120,62 +113,28 @@ async function claimPointFromInvite({
 
   const inviteAddress = inviteWallet.address;
 
-  progress(TRANSACTION_STATES.GENERATING);
-
-  // transfer from invite wallet to new wallet
-  const transferTx = azimuth.ecliptic.transferPoint(
-    contracts,
-    point,
-    wallet.ownership.keys.address
-  );
-  transferTx.gas = 500000; //TODO can maybe be lower?
-
-  // ping gas tank with txs if needed
-  let inviteNonce = await web3.eth.getTransactionCount(inviteAddress);
-  transferTx.nonce = inviteNonce++;
-  transferTx.gasPrice = 20000000000; //NOTE we pay the premium for faster ux
-  transferTx.from = inviteAddress;
-
-  progress(TRANSACTION_STATES.SIGNING);
-  // NOTE: using web3.eth.accounts.signTransaction is broken (1.0.0-beta51)
-  const transferStx = new Tx(transferTx);
-  transferStx.sign(inviteWallet.privateKey);
-  const rawTransferStx = hexify(transferStx.serialize());
-  const transferCost = transferTx.gas * transferTx.gasPrice;
-
-  progress(TRANSACTION_STATES.FUNDING_INVITE);
-
-  let usedTank = await tank.ensureFundsFor(
-    web3,
-    point,
-    inviteAddress,
-    transferCost,
-    [rawTransferStx],
-    askForFunding,
-    gotFunding
-  );
-
-  progress(TRANSACTION_STATES.CLAIMING);
-
-  // send transaction
-  await sendAndAwaitConfirm(web3, [Just(transferStx)], usedTank);
-
   //
-  // we're gonna be operating as the new wallet from here on out, so change
-  // the relevant values
+  // generate transactions
   //
-  const newAddress = wallet.ownership.keys.address;
 
-  // configure management proxy
-  const managementTx = azimuth.ecliptic.setManagementProxy(
+  progress(TRANSACTION_STATES.GENERATING);
+
+  // transfer to invite wallet, so it can configure
+  //NOTE doing this in the invite wallet instead of the final wallet makes
+  //     failure cases "solvable" by just telling the user to fully retry
+
+  //TODO no harm done if we already owned it, but should still get a bool arg
+  //     for skipping this, if it isn't too big a burden for callers
+  let transferTmpTx = azimuth.ecliptic.transferPoint(
     contracts,
     point,
-    wallet.management.keys.address
+    inviteAddress,
+    false
   );
-  managementTx.gas = 200000;
-  managementTx.nonce = 0;
+  transferTmpTx.gas = 500000; //TODO can maybe be lower?
```

Comment:
can we make this `toWei('500', 'gwei')` or whatever the more semantic value is? the magic numbers for gas will definitely confuse readers in the future.

Reply:
(let's just update the magic gas numbers in this file only, for now?) or at least make them constants like `TRANSFER_TX_GAS` or something.

Reply:
These aren't prices, but limits. Gwei doesn't exist in this context. Strong agree on the constants though. I should probably do some transaction analysis to actually find these values.

Reply:
oh shit yeah totally 😅